.github/workflows/container.yml

```
@@ -79,7 +83,7 @@ jobs:
     - name: Upload RPM as artifact
       uses: actions/upload-artifact@v4
       with:
-        name: epics-bundle-rpm
+        name: epics-bundle-rpm-alma${{ matrix.alma_version }}
```

```suggestion
        name: epics-bundle-rpm-el${{ matrix.alma_version }}
```
Use `el` instead of `alma` since that is the standard for specifying `rhel`- like.